ScrapPackedExplorerCli/CliOptions.cs

```diff
@@ -62,14 +64,41 @@ class ExtractOptions : BaseOptions
     [Verb("list", HelpText = "list or search files and folders in the archive")]
     class ListOptions : BaseOptions
     {
-        [Option('l', "outputStyle", Required = false, Default = "list", HelpText = "Output list (default) or tree view")]
-        public string outputStyle { get; set; }
+        [Option('l', "outputStyle", Required = false, Default = OutputStyles.List, HelpText = "Output list (default) or tree view")]
+        public OutputStyles outputStyle { get; set; }
 
         [Option('q', "searchString", Required = false, Default = "", HelpText = "A Search string to filter the output with")]
         public string searchString { get; set; }
 
-        [Option('r', "regex", Required =false, Default = false, HelpText = "Defines if the search string is a regular expression")]
+        [Option('r', "regex", Required = false, Default = false, HelpText = "Defines if the search string is a regular expression")]
         public bool isRegex { get; set; }
+
+        // todo: come up with better description
+        // todo: change arguments style
+        [Option('s', "starts-with", Required = false, Default = false, HelpText = "Apply search query only to beginnng of the files path \nBy default applies everywhere")]
+        public bool StartsWith { get; set; }
+
+        [Option('f', "match-filename", Required = false, Default = false, HelpText = "Search only by files. \nBy default search includes folders")]
+        public bool MatchFilename { get; set; }
+
+        // todo: change short arguments
+        [Option('z', "show-file-size", Required = false, Default = false, HelpText = "Show files sizes")]
+        public bool ShowFileSize { get; set; }
+
+        [Option('x', "show-file-offset", Required = false, Default = false, HelpText = "Show files offsets")]
+        public bool ShowFileOffset { get; set; }
+
+        // todo: make custom helper to make better help screen
+        // note: https://github.com/commandlineparser/commandline/wiki/HelpText-Configuration
+    }
+
+    [Flags]
+    public enum OutputStyles
+    {
+        None = 0x0,
+        List = 0x1,
+        Tree = 0x2,
+        Name = 0x3
```

Comment:
I thought it would be eaaier to do like this. Why not? Enums are cool

Reply:
don't get me wrong … i'm not against enums …
i just think the `= 0x?` is not needed

Reply:
Well, can be just 1, 2, 4... etc.
But it should be powers of two because `[Flag]`. It makes bitmask flag from enum, and if things go about bits I like to make some visual notes like `0x` and stuff

Reply:
ah ok … yea didn't see the `[Flag]`
without it i'd have preffered
```C#
public enum OutputStyles
    {
        None,
        List,
        Tree,
        Name
}
```
because why define numbers when numbers are not needed …
but with `[Flag]` its good this way